libkbfs/data_types.go

```diff
@@ -628,6 +628,9 @@ type EntryInfo struct {
 	Mtime int64
 	// Ctime is in unix nanoseconds
 	Ctime int64
+	// If this is a team TLF, we want to track the last writer of an
+	// entry, since in the block, only the team ID will be tracked.
+	TeamWriter keybase1.UID `codec:"tw,omitempty"`
```

Comment:
Do we maybe want this to be `Writer keybase.UID`? In other words, use this for both team folders and user folders so that 1) we can use the same code path when we get latest writer; 2) when we eventually upgrade to the same design for non-team TLFs this will work for that too.